### ISSUE TITLE
After completing one request, debug session does not reload correct context for remaining request(s)

### ISSUE THREAD

**Issue by mcdado** [xdebug/vscode-php-debug#310]
PHP version:  tested 7.0.32 and 7.1.22
XDebug version:  2.6.1
Adapter version:

Your launch.json:  
```json-with-comments
{
    // Use IntelliSense to learn about possible Node.js debug attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Listen for XDebug",
            "stopOnEntry": true,
            "type": "php",
            "request": "launch",
            "port": 9001,
            "xdebugSettings": {
              "max_data": 4096,
              "max_children": 128
            }
        },
        {
            "name": "Launch currently open script",
            "type": "php",
            "request": "launch",
            "program": "${file}",
            "cwd": "${fileDirname}",
            "port": 9001
        }
    ]
}
```
XDebug php.ini config:
```
zend_extension="/usr/local/lib/php/pecl/20151012/xdebug.so"
xdebug.remote_enable = 1
xdebug.remote_port = 9001
xdebug.var_display_max_data = 4096
xdebug.var_display_max_depth = 8
```

XDebug logfile (from setting `xdebug.remote_log` in php.ini):  
Adapter logfile (from setting `"log": true` in launch.json):

Steps to reproduce:

See it in action: https://cl.ly/fd0c09cccb46/vscode-php-debug-report.mp4

**NB**: For simplicity's sake I enabled "stopOnEntry", when the mouse goes off screen to the left I'm launching new requests from the browser, which break right away. You could even run `curl http://127.0.0.1/?XDEBUG_SESSION_START=1 &` in your integrated terminal for launching a bunch of requests.

Basically, when you go complete running a request and this one disappears from the Call Stack, the context of other requests is not reloaded, meaning you cannot continue the, because even though they're "paused on entry" the play button is not enabled, the state is out of sync I guess. I hope this is clear enough from my side to be understandable. This happens with more than two requests, too.

**Comment by mcdado** [comment]
Maybe related: 
https://github.com/felixfbecker/vscode-php-debug/issues/301
https://github.com/felixfbecker/vscode-php-debug/issues/231
https://github.com/felixfbecker/vscode-php-debug/issues/167

**Comment by zobo** [comment]
Not reproduceable anymore.
